build_integration.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-: ${ALL_RELEASES:="0.8.2.2 0.9.0.1 0.10.1.1 0.10.2.2 0.11.0.3 1.0.2 1.1.1 2.0.1 2.1.1 2.2.1 2.3.0 2.4.0 2.5.0"}
+: ${ALL_RELEASES:="0.8.2.2 0.9.0.1 0.10.1.1 0.10.2.2 0.11.0.3 0.11.0.2 0.11.0.1 1.0.2 1.1.1 2.0.1 2.1.1 2.2.1 2.3.0 2.4.0 2.5.0"}
```

Comment:
are there big changes btwn these additional releases? I typically would only test against the latest patch release unless there is a known api change.

Reply:
I forgot to remove those versions, thanks.

Reply:
Out of scope for this PR, but if we're starting to revive this project, I'd be 👍 on a new major release that drops support for really old versions of Kafka. Those folks can keep with their old versions of `kafka-python`, but it'd let us cleanup the codebase by removing a bunch of special cases, especially the code that tries to infer the protocol version because it wasn't yet available through the API.

Reply:
For sure. I was hoping we could keep a list of tracked/supported Kafka releases in one place inside this repo, which could propagate forward into the GitHub workflow matrix. I haven't decided how to go about that yet, and I think it belongs in a separate PR.